addons/survey/wizard/survey_invite.py

```diff
@@ -66,6 +65,16 @@ def _get_default_author(self):
     survey_users_login_required = fields.Boolean(related="survey_id.users_login_required", readonly=True)
     survey_users_can_signup = fields.Boolean(related='survey_id.users_can_signup')
     deadline = fields.Datetime(string="Answer deadline")
+    send_email = fields.Boolean(compute="_compute_send_email",
+                                inverse="_inverse_send_email")
+
+    @api.depends('survey_access_mode')
+    def _compute_send_email(self):
+        for record in self:
+            record.send_email = record.survey_access_mode == 'token'
```

Comment:
Seems different from your context usage, where you set `!= public` (which seems better).

Actually, does this compute/onchange do anything?
Maybe it would help getting rid of "default_send_email" context?

Reply:
You don't need to set an inverse method, you can just set the attribute "readonly=False" on the field definition (and optionally "stored=True" if you want it to be stored as computed field are not stored by default).

You could try to make the compute method depend on the "survey" field rather so that if the survey is changed it changes the value of "send_email" according to that survey permission.
Even though I don't think there is a way to modify the survey from the wizard itself?

So maybe just keeping the context to set the default value is sufficient in this case, and it would simplify by removing the compute method.

I'm not entirely sure, both would be ok, but when I hesitate I tend to vote for the more simple/short solution, which is just having the context and no compute.